projects.yaml

```diff
@@ -1,35 +1,42 @@
-projects:
-    - ['https://github.com/mattvenn/wrapped_rgb_mixer', 'mpw3']
-    - ['https://github.com/mattvenn/wrapped_frequency_counter', 'mpw3']
+required_interfaces:
+    power: {vccd1: 1, vssd1: 1}
+    active: {active: 1}
 
 interfaces:
-    required:
-        power : {vccd1: 1, vssd1: 1}
-        clock : {wb_clk_i: 1}
-        active: {active: 1}
-
-    optional:
-        gpio  : {io_in: 38, io_out: 38, io_oeb: 38}
-        la    : {la_data_in: 32, la_data_out: 32, la_oenb: 32}
-        irq   : {irq: 3}
-        clk2  : {user_clock2: 1}
-        wb    : {wb_rst_i: 1, wb_stb_i: 1, wb_cyc_i: 1, wbs_we_i: 1, wbs_sel_i: 4, wbs_dat_i: 32, wbs_addr_i: 32, wbs_ack_o: 1, wbs_dat_o: 32}
```

Comment:
these yaml style will now break the port validation added on my last commit. 
I do'nt have a strong preference which way it's done, but do think that clk should probably be required